AsyncDisplayKit/ASCellNode.mm

```diff
@@ -211,6 +211,10 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
 
 #pragma clang diagnostic pop
 
+- (void)applyLayoutAttributes:(UICollectionViewLayoutAttributes *)layoutAttributes
+{
+}
```

Comment:
Can you add a comment why this empty and add the purpose for it to the header file. Here could be something like:  `// To be overriden by subclasses`